app/views/recipe_foods/new.html.erb

```diff
@@ -0,0 +1,32 @@
+<%= form_with model: @recipe_food, url: recipe_foods_path, method: :post, local: true, scope: :recipe_food do |form| %>
+  <% if @recipe_food.errors.any? %>
+    <div style="color: red">
+      <h2><%= pluralize(@recipe_food.errors.size, "error") %> prohibited this food from being saved:</h2>
+
+      <ul>
+        <% @recipe_food.errors.each do |error| %>
+          <li><%= error.full_message %></li>
+        <% end %>
+      </ul>
+    </div>
+  <% end %>
+
+  <div class="form-group">
+    <%= form.label :quantity %>
+    <%= form.text_field :quantity, class:"form-control" %>
+  </div>
+
+  <div class="form-group">
+    <%= form.label :recipe_id %>
+    <%= form.text_field :recipe_id, class:"form-control" %>
+  </div>
+
+  <div class="form-group">
+    <%= form.label :food_id %>
+    <%= form.text_field :food_id, class:"form-control" %>
+  </div>
+
+  <div>
+    <%= form.submit class:"btn btn-secondary" %>
+  </div>
+<% end %>

```

Comment:
Hello @KDlamini, at my end adding a recipe is working.

Reply:
@ridwanediallo in that case, the next reviewer will double-check it. Perhaps they will have a different result.

I just checked again, nothing is happening on my end. But as I explained above maybe it's being added I cannot see it since the table is being blocked by the footer and scrolling is not implemented either. However, the shopping list remains empty after adding ingredients on my end.

If you are confident that it's working, the next reviewer will verify that.